addon/components/dialog.ts

```diff
@@ -10,7 +10,7 @@ import { modifier } from 'ember-modifier';
 
 import type DialogStackProvider from 'ember-headlessui/services/dialog-stack-provider';
 
-function getPortalRoot() {
+export function getPortalRoot() {
```

Comment:
This function has been exported so that the test code can target this element in the exact same way. Sharing the "lib" code with the tests means that we can change them both together if the root element changes somewhere down the road.

Reply:
should we mark all exported functions not intended for public consumption with `// @private`?

Reply:
Great idea! I'll do that 👍

Reply:
🙌